<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HTMLConverter</class>
 <widget class="QWidget" name="HTMLConverter">


  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HTML Converter</string>
  </property>


  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">

    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="path_line_edit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>/path/to/a/text/file</string>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QPushButton" name="browse_btn">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>

    <!-- convert button -->
    <item>
      <widget class="QPushButton" name="convert_btn">
       <property name="text">
        <string>Convert</string>
       </property>
      </widget>
     </item>
    </layout>

   </item>


   <item row="1" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>92</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>124</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>


    <item row="2" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">

                <!--the content of the file that has been selected-->
            <item>
                <widget class="QLabel" name="file_content">
                    <property name="text">
                        <string></string>
                    </property>
                </widget>
            </item>

        </layout>
    </item>


   <item row="2" column="2">
    <layout class="QVBoxLayout" name="verticalLayout">


    <!-- <item> -->
      <!-- <widget class="QPushButton" name="push_btn">
        <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>50</horstretch>
                <verstretch>100</verstretch>
            </sizepolicy>
        </property>

        <property name="minimumSize">
            <size>
                <width>120</width>
                <height>20</height>
            </size>
        </property>

        <property name="maximumSize">
            <size>
                <width>120</width>
                <height>20</height>
            </size>
        </property>

        <property name="text">
            <string>Push Button</string>
        </property>

      </widget>
     </item> -->


    <!-- </layout> -->
   <!-- </item> -->


    <!-- <item row="3" column="2"> -->
        <!-- <layout class="QVBoxLayout" name="verticalLayout"> -->
    
            <!-- <item>
                <widget class="QPushButton" name="text_edit">
                <property name="text">
                    <string>Text Edit</string>
                </property>
                <property name="geometry">
                    <width>120</width>
                    <height>0</height>
                </property>

                <property name="minimumSize">
                    <size>
                        <width>120</width>
                        <height>20</height>
                    </size>
                </property>

                <property name="maximumSize">
                    <size>
                        <width>120</width>
                        <height>20</height>
                    </size>
                </property>

                </widget>
            </item> -->


        </layout>
    </item>

   <item row="2" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>335</width>
       <height>100</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>


 </widget>
 <resources/>
 <connections/>
</ui>
